source/_integrations/dio_chacon.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Dio Chacon
+description: Instructions on how to integrate your Dio Chacon devices within Home Assistant.
+ha_category:
+  - Cover
+ha_release: 2024.5
+ha_iot_class: Cloud Push
+ha_config_flow: true
+ha_codeowners:
+  - '@cnico'
+ha_domain: dio_chacon
+ha_platforms:
+  - cover
+ha_integration_type: integration
+---
+
+[Dio Chacon devices](https://chacon.com/en/) are connected home devices that can be controlled via RF 433 Mhz or Wifi.
+This integration gives you access to the Wifi connection that allows in Home Assistant to list your devices and interact with them in real time like vendor's smartphone application does.
+
+There is currently support for the following information within Home Assistant:
+
+- Cover devices (get statuses, moves up, down, stop and moves to a given percentage)
+
+## Prerequisites
+
+You will need to use the standalone app for this device to register a username and password.
+
+- [Google](https://play.google.com/store/apps/details?id=com.chacon.dioone)
+- [Apple](https://apps.apple.com/fr/app/dio-one/id1493503504)
```

Comment:
This link points to the French app store. Is is available in France only? Otherwise, it should be in English.

Reply:
I updated the links to point to english versions of the stores pages.
I do not think there are differents applications per country.